core/worker/src/main/java/com/cognifide/aet/worker/drivers/WebDriverProvider.java

```diff
@@ -23,45 +23,46 @@
 import com.google.common.base.Joiner;
 import com.google.common.collect.Maps;
 import java.util.Map;
-import org.apache.felix.scr.annotations.Activate;
-import org.apache.felix.scr.annotations.Component;
-import org.apache.felix.scr.annotations.Properties;
-import org.apache.felix.scr.annotations.Property;
-import org.apache.felix.scr.annotations.Reference;
-import org.apache.felix.scr.annotations.ReferenceCardinality;
-import org.apache.felix.scr.annotations.ReferencePolicy;
-import org.apache.felix.scr.annotations.Service;
+
 import org.osgi.framework.Constants;
+import org.osgi.service.component.annotations.*;
+import org.osgi.service.metatype.annotations.Designate;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
  * @author lukasz.wieczorek
  */
-@Service(WebDriverProvider.class)
-@Component(label = "AET WebDriver Provider", description = "AET WebDriver Provider", immediate = true, metatype = true)
-@Properties({@Property(name = Constants.SERVICE_VENDOR, value = "Cognifide Ltd")})
+@Component(
+    service = WebDriverProvider.class,
+    property = {"name = " + Constants.SERVICE_VENDOR, "value = Cognifide Ltd"},
+    immediate = true
+)
+@Designate(ocd = WebDriverProviderConf.class)
 public class WebDriverProvider {
 
   private static final Logger LOG = LoggerFactory.getLogger(WebDriverProvider.class);
 
-  private static final String DEFAULT_WEB_DRIVER_NAME = "defaultWebDriverName";
-
-  @Property(name = DEFAULT_WEB_DRIVER_NAME, label = "Default Web Driver name", value = "ff")
-  private String defaultWebDriverName;
+  private WebDriverProviderConf webDriverProviderConf;
 
-  @Reference(referenceInterface = WebDriverFactory.class, policy = ReferencePolicy.DYNAMIC, cardinality = ReferenceCardinality.OPTIONAL_MULTIPLE, bind = "bindWebDriverFactory", unbind = "unbindWebDriverFactory")
+  @Reference(
+      service = WebDriverFactory.class,
+      policy = ReferencePolicy.DYNAMIC,
+      cardinality = ReferenceCardinality.MULTIPLE,
+      bind = "bindWebDriverFactory",
+      unbind = "unbindWebDriverFactory")
   private final Map<String, WebDriverFactory> collectorFactories = Maps.newConcurrentMap();
 
   @Reference
   private ProxyServerProvider proxyServerProvider;
 
   @Activate
-  void activate(Map<String, String> properties) {
-    defaultWebDriverName = properties.get(DEFAULT_WEB_DRIVER_NAME);
+  void activate(WebDriverProviderConf webDriverProviderConf) {
+    this.webDriverProviderConf = webDriverProviderConf;
   }
 
-  public WebCommunicationWrapper createWebDriverWithProxy(String preferredWebDriver, String proxyName)
+  public WebCommunicationWrapper createWebDriverWithProxy(String preferredWebDriver,
```

Comment:
For the future could you please don't change formatting on the lines that didn't change during the refactoring?
Please create additional PR to fix only formatting.